tiledb/sm/cache/lru_cache.h

```diff
@@ -144,17 +143,15 @@ class LRUCache {
    * @param size The logical size of the object.
    * @param overwrite If `true`, if the object exists in the cache it will be
    *     overwritten. Otherwise, the new object will be deleted.
-   * @return Status
    */
-  Status insert(
-      const K& key, V&& object, uint64_t size, bool overwrite = true) {
+  void insert(const K& key, V&& object, uint64_t size, bool overwrite = true) {
     // Do nothing if the object size is bigger than the cache maximum size
     if (size > max_size_)
-      return Status::Ok();
+      return;
```

Comment:
Should put curly braces around if-block.
I'm not requesting a change because of this.